README.md

```diff
@@ -6,6 +6,8 @@ This [JavaScript GitHub Action](https://help.github.com/en/actions/building-acti
 A workaround is to use a [personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) from a [personal user/bot account](https://help.github.com/en/github/getting-started-with-github/types-of-github-accounts#personal-user-accounts).
 However, for organizations, GitHub Apps are [a more appropriate automation solution](https://developer.github.com/apps/differences-between-apps/#machine-vs-bot-accounts).
 
+If using GitHub Enterprise, set the `GITHUB_API_URL` environment variable to override the default API URL.
```

Comment:
I don't think a user would need to set this manually when calling the action. I think this is just a default environment variable inside GHE.

https://docs.github.com/en/enterprise-server@3.1/actions/reference/environment-variables

> GITHUB_API_URL | Returns the API URL. For example: http(s)://[hostname]/api/v3.

I don't run GHE so I can't test this to make 100% sure.

Reply:
I just checked, yup, it does output the right value.  Removing the changes on README.md.